app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivity.java

```diff
@@ -88,22 +95,26 @@ public void setTimeZone(DateTimeZone zone) {
 
     @Override
     @DynamoDBIgnore
+    @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getScheduledOn() {
         return getInstant(getLocalScheduledOn());
     }
 
     @Override
+    @JsonDeserialize(using = DateTimeDeserializer.class)
     public void setScheduledOn(DateTime scheduledOn) {
         this.localScheduledOn = (scheduledOn == null) ? null : scheduledOn.toLocalDateTime();
     }
 
     @Override
     @DynamoDBIgnore
+    @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getExpiresOn() {
         return getInstant(getLocalExpiresOn());
     }
 
     @Override
+    @JsonDeserialize(using = DateTimeDeserializer.class)
```

Comment:
The unit tests don't test deserializing scheduledOn or expiresOn. Can we add that to the unit tests?